src/debug/ee/debugger.cpp

```diff
@@ -2113,18 +2113,9 @@ HRESULT Debugger::Startup(void)
             ShutdownTransport();
             ThrowHR(hr);
         }
-
     #ifdef FEATURE_PAL
         PAL_SetShutdownCallback(AbortTransport);
     #endif // FEATURE_PAL
-
-         bool waitForAttach = CLRConfig::GetConfigValue(CLRConfig::UNSUPPORTED_DbgWaitForDebuggerAttach) != 0;
```

Comment:
Does this edit relate to the bug fix? I'm not opposed to cleanup, but I don't know where the bar is now. If this is an unrelated code change you might be asked to split it out into a seperate commit that won't get integrated to RTM.